content/doc/book/managing/change-system-timezone.adoc

```diff
@@ -42,26 +42,21 @@ where TZ is a java.util.TimeZone ID ("Europe/Paris" for example).
 
 _Note that `+user.timezone=Europe/Paris+` will work as well, but it can interfere with other contexts._
 
-If running Jenkins via a system package, this can be accomplished by
-setting JAVA_ARGS in your /etc/default/jenkins (Debian and Ubuntu):
+If running Jenkins via a Linux package, this can be accomplished
+by adding the following to `/etc/systemd/system/jenkins.service.d/override.conf`:
```

Comment:
Would it help users that are unfamiliar with `systemd` configuration editing to have the command suggested to them?
```suggestion
by adding the following to `/etc/systemd/system/jenkins.service.d/override.conf` (for example, using `systemd edit jenkins`):
```

Reply:
Done in commit 4b280c8.